From c9bcb69d41408407e408f93f39913e9dd084fcbc Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Fri, 17 Dec 2021 17:16:14 -0500
Subject: [PATCH 30/32] Reduce the size of udfps enroll progress bar. Some
 devices have their udfps pretty low, and the progress bar would make the icon
 at the wrong place

Change-Id: I1609ad9ca316293dcaaf07f7e681d11aadfcd29c
---
 packages/SystemUI/res/values/config.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/res/values/config.xml b/packages/SystemUI/res/values/config.xml
index 7ed915600230..4e50f1b470a2 100644
--- a/packages/SystemUI/res/values/config.xml
+++ b/packages/SystemUI/res/values/config.xml
@@ -577,7 +577,7 @@
 
     <!-- The radius of the enrollment progress bar, in dp -->
     <integer name="config_udfpsEnrollProgressBar" translatable="false">
-        280
+        70
     </integer>
 
     <!-- package name of a built-in camera app to use to restrict implicit intent resolution
-- 
2.32.1 (Apple Git-133)

